Task_HelloWorld/index.js

```diff
@@ -1,26 +1,42 @@
-// Print hello World
-// Write your code here
-
-// Write your code above
-
-//Declare a variable i and set its value to 5
-// Write your code here
-
-// Write your code above
-
-//Write the if else statement to console "Number is odd" if the value of i is odd and "Number is even" if the value of i is even.
-// Write your code here
-
-// Write your code above
-
-//Declare a string variable which is always constant and set is value to 'triangle'.
-// Write your code here
-
-// Write your code above
-
-//Write a function 'findSide' which takes an argument 'type' and return the number of sides in that shape
-// Write your code here
-
-// Write your code above
-const side = findSide('Circle');
+// Print hello World
+// Write your code here
+console.log('Hello World');
+// Write your code above
+
+//Declare a variable i and set its value to 5
+// Write your code here
+let i = 5;

```

Comment:
It may be clearer to return a string like 'Cannot determine' instead of 'Infinite' for the 'Circle' case in 'findSide' function.